File: ui/src/main/java/com/midtrans/sdk/corekit/models/ItemDetails.java
package com.midtrans.sdk.corekit.models;

/**
 * It hold an information about purchased item, </p>like id, price etc.
 *
 * Created by shivam on 10/29/15.
 */
public class ItemDetails extends com.midtrans.sdk.uikit.api.model.ItemDetails {
    public ItemDetails(String id, Double price, Integer quantity, String name) {
        super (id, price, quantity, name);
    }
}

File: ui/src/main/kotlin/com/midtrans/sdk/uikit/api/model/ItemDetails.kt
package com.midtrans.sdk.uikit.api.model

import com.midtrans.sdk.corekit.api.model.ItemDetails

open class ItemDetails(
    id: String? = null,
    price: Double? = null,
    quantity: Int = 0,
    name: String? = null,
) : ItemDetails(
    id = id,
    price = price,
    quantity = quantity,
    name = name
)